bin/varnishtest/tests/b00099.vtc

```diff
@@ -0,0 +1,19 @@
+varnishtest "Test varnishadm -j option"
+
+feature ignore_unknown_macro
+
+feature cmd {python --version -m json.tool 2>&1 | grep -q 'Python [0-9].[0-9]'}
+
+varnish v1 -vcl { backend foo { .host = "127.0.0.1"; } } -start
+
+shell -expect "VALID" {
+	${topbuild}/bin/varnishadm/varnishadm -n ${v1_name} help -j | python -m json.tool >> /dev/null && echo "VALID" || echo "NOT"
+}
+
+shell -expect "VALID" {
+	${topbuild}/bin/varnishadm/varnishadm -n ${v1_name} backend.list -j | python -m json.tool >> /dev/null && echo "VALID" || echo "NOT"
+}
+
+shell -expect "VALID" {
+	${topbuild}/bin/varnishadm/varnishadm -n ${v1_name} ping -j | python -m json.tool >> /dev/null && echo "VALID" || echo "NOT"
+}
```

Comment:
You can drop the topbuild from all these.

Reply:
Thanks, have dropped as you suggested.